domain-models-runtime/pom.xml

```diff
@@ -357,6 +358,22 @@
               </resources>
             </configuration>
           </execution>
+          <execution>
+            <id>copy-ramls</id>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${basedir}/target/classes/ramls</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>${ramlfiles_path}</directory>
+                  <filtering>true</filtering>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
```

Comment:
Thank you for the unit tests!

The raml directory with test.raml and test.schema ends up in the .jar file. But other modules that use RMB do not need them because they are for RMB unit tests only.
My suggestion is to move this raml directory from domain-models-runtime/raml to domain-models-runtime/src/test/resources/raml, and to replace ${basedir}/target/classes/ramls by ${basedir}/target/test-classes/ramls.

Reply:
That makes sense.